cns/cnsclient/apiclient.go

```diff
@@ -4,5 +4,6 @@ import "github.com/Azure/azure-container-networking/cns"
 
 // APIClient interface to update cns state
 type APIClient interface {
-	UpdateCNSState(createNetworkContainerRequest *cns.CreateNetworkContainerRequest) error
+	CreateOrUpdateNC(*cns.CreateNetworkContainerRequest) error
+	InitCNSState(*cns.CreateNetworkContainerRequest, map[string]*cns.KubernetesPodInfo) error
```

Comment:
nit: can you reverse the order of these APIs. Like InitCnsState before CreateOrUpdateNetworkContainer